ossindex/ossindex.go

```diff
@@ -170,3 +167,17 @@ func AuditPackages(purls []string) ([]types.Coordinate, error) {
 	}
 	return results, nil
 }
+
+func setupRequest(jsonStr []byte) (req *http.Request, err error) {
+	if req, err = http.NewRequest(
+		"POST",
+		getOssIndexUrl(),
+		bytes.NewBuffer(jsonStr)); err != nil {
```

Comment:
This is nitpicky and it was there already, but I'm really not a fan of the inline if for something that spans multiple lines, took me a second to realize where the request builder ended and it went back to the `if`

Reply:
Done in: 72e7291